src/PostgREST/QueryBuilder.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE TupleSections #-}
 module PostgREST.QueryBuilder
```

Comment:
I believe that if we merge ```QueryBuilder``` and ```PgQuery``` we can expose less functions and better encapsulate these modules. But to better reason about this we also need an explicit import list in the App module.